packages/textlint/src/options.ts

```diff
@@ -1,11 +1,11 @@
 // LICENSE : MIT
 "use strict";
-import { getFormatterList } from "textlint-formatter";
+import { getFormatterList, FormatterDetail } from "textlint-formatter";
 import { getFormatterList as getFixerFormatterList } from "textlint-fixer-formatter";
 
 const optionator = require("optionator");
 
-const concatFormatterList = (formatterList: { name: string }[]) => {
+const concatFormatterList = (formatterList: FormatterDetail[]) => {
```

Comment:
`concatFormatterList()` should accept both `FormatterDetail[]` and `FixerFormatterDetail[]`? e.g.

```diff
diff --git a/packages/textlint/src/options.ts b/packages/textlint/src/options.ts
index 33a1017..62a0e0d 100644
--- a/packages/textlint/src/options.ts
+++ b/packages/textlint/src/options.ts
@@ -1,11 +1,11 @@
 // LICENSE : MIT
 "use strict";
-import { getFormatterList } from "textlint-formatter";
-import { getFormatterList as getFixerFormatterList } from "textlint-fixer-formatter";
+import { getFormatterList, FormatterDetail } from "textlint-formatter";
+import { getFormatterList as getFixerFormatterList, FixerFormatterDetail } from "textlint-fixer-formatter";

 const optionator = require("optionator");

-const concatFormatterList = (formatterList: { name: string }[]) => {
+const concatFormatterList = (formatterList: FormatterDetail[] | FixerFormatterDetail[]) => {
     return formatterList
         .map(formatter => {
             return formatter.name;
```

Reply:
Nice catch!

Fixed in https://github.com/textlint/textlint/pull/429/commits/c27bacc98420ca48d83dd44038749b3a14c356e5